packages/wdio-appium-service/src/launcher.ts

```diff
@@ -17,34 +17,68 @@ const DEFAULT_CONNECTION = {
 }
 
 export default class AppiumLauncher {
-    constructor(options, capabilities, config) {
-        this.options = options
-        this.capabilities = capabilities
+    public logPath: string
+    public command: string
+    public appiumArgs: Array<string> = []
+    public capabilities: Array<AppiumCapability>
+    public process?: ChildProcessByStdio<null, Readable, Readable>
+    private readonly basePath: string = '/'
+    public args: KeyValueArgs | ArgValue[]
+
+    constructor(
+        private options: AppiumOptions,
+        capabilities: Array<AppiumCapability> | AppiumCapability = {},
+        public config?: Config
+    ) {
+        /**
+         * Convert capability object to Array of capabilities
+         */
+        this.capabilities = Array.isArray(capabilities)
+            ? capabilities
+            : Object.values(capabilities)
         this.args = {
-            basePath: '/',
-            ...(options.args || {})
+            basePath: this.basePath,
+            ...(this.options.args || {})
         }
-        this.logPath = options.logPath || config.outputDir
+        this.logPath = options.logPath || config?.outputDir
```

Comment:
Currently in all examples logPath is being passed as `logPath : './'`

https://github.com/webdriverio/webdriverio/tree/master/packages/wdio-appium-service#logpath

I consider to add it's as a default value, because it's been required in `redirectLogStream` AppiumLauncher method.

Wondering what is your opinion on it?

Reply:
if `logPath` and `outputDir` are not set it should print Appium logs to stdout.

Reply:
fixed here https://github.com/webdriverio/webdriverio/pull/6149/commits/f799adc68697afd3c57aaa83b9ab445fc12ada24